en/advanced/gimbal_control.md

```diff
@@ -27,26 +27,38 @@ The output is set using the [MNT_MODE_OUT](../advanced_config/parameter_referenc
+The ground station sends high level [MAVLink Gimbal Manager](https://mavlink.io/en/services/gimbal_v2.html#gimbal-manager-messages) commands to the manager of the gimbal it wants to control, and the manager will in turn send appropriate lower level "gimbal device" commands to control the gimbal.
+
+PX4 can be configured as the gimbal manager to control a single gimbal device (which can either be physically connected or be a MAVLink gimbal that implements the [gimbal device interface](https://mavlink.io/en/services/gimbal_v2.html#gimbal-device-messages)).
+
+To enable a MAVLink gimbal, first set parameter [MNT_MODE_IN](../advanced_config/parameter_reference.md#MNT_MODE_IN) to `MAVlink gimbal protocol v2` and [MNT_MODE_OUT](../advanced_config/parameter_reference.md#MNT_MODE_OUT) to `MAVLink gimbal protocol v2`.
 
 The gimbal can be connected to *any free serial port* using the instructions in [MAVLink Peripherals (GCS/OSD/Companion)](../peripherals/mavlink_peripherals.md) (also see [Serial Port Configuration](../peripherals/serial_configuration.md#serial-port-configuration)).
 For example, if the `TELEM2` port on the flight controller is unused you can connect it to the gimbal and set the following PX4 parameters:
 - [MAV_1_CONFIG](../advanced_config/parameter_reference.md#MAV_1_CONFIG) to **TELEM2** (if `MAV_1_CONFIG` is already used for a companion computer (say), use `MAV_2_CONFIG`).
 - [MAV_1_MODE](../advanced_config/parameter_reference.md#MAV_1_MODE) to **NORMAL**
 - [SER_TEL2_BAUD](../advanced_config/parameter_reference.md#SER_TEL2_BAUD) to manufacturer recommended baud rate.
 
-With this setup, ground stations can send [MAVLink Gimbal Commands](https://mavlink.io/en/services/gimbal_v2.html#gimbal-manager-messages) to PX4, such as `MAV_CMD_DO_SET_ROI_LOCATION` to track region of interest and `MAV_CMD_DO_GIMBAL_MANAGER_PITCHYAW` to control the gimbal attitude, and so on.
+### Multiple Gimbal Support
+
+PX4 does not automatically create gimbal managers for other gimbal devices on the system.
+
+You can support additional gimbals provided that they:
+
+- implement the gimbal _manager_ protocol
+- Are visible to the ground station and PX4 on the MAVLink network.
+  This may require that traffic forwarding be configured between PX4, the GCS, and the gimbal.
+- Each gimbal must have a unique component id.
+  For a PWM connected gimbal this will be the component ID of the autopilot
 
-:::note
-In Gimbal Protocol v2 terms, this setup configures PX4 as a _Gimbal Manager_.
-The flight stack handles high level "gimbal manager" commands and mission items, sending appropriate lower level "gimbal device" commands to control the gimbal.
-:::
+Note that MAVLink does not support control of individual cameras in missions.
+Therefore any command sent in a mission will be broadcast to all connected cameras.
```

Comment:
I don't think that's true. You can specify the gimbal device in the commands. E.g. in [MAV_CMD_DO_GIMBAL_MANAGER_PITCHYAW](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_GIMBAL_MANAGER_PITCHYAW) you can set the "Gimbal device ID" in `param7`.

Reply:
Right, so you're saying you would want some sort of gimbal ID, rather than a component ID. And the reasoning is that you don't know the gimbals connected when you plan a mission offline. That's true but I would argue that you also don't know the gimbal IDs essentially, you just don't really know without connecting.

Reply:
Good point.

A gimbal id would be slightly better than a component ID for the limited case where you can assume all gimbal payloads are the same. IN this case you could set the ID for each gimbal without having to know what cameras are attached, and create a generic mission that would work on any vehicle that had three cameras.

All a bit abstract though, since something still needs to map ids to gimbals etc.

If gimbal payloads might be different then you might be able to specify by requirement on the camera - i.e. ROI command to go to gimbals that support IR cameras, or whatever. That seems like something  you might want, and we could think about in the far future.

Upshot. I have convinced myself that IDs make enough sense for now, and we have to live with non-generic missions until we have a stakeholder who sees value in hammering this out.